### ISSUE TITLE
Gene Symbols and SemMedDB

### ISSUE THREAD

**Issue by karafecho** [NCATSTranslator/testing#229]
Issues with gene symbols surfaced yet again with this month’s challenge question (QotM #5), with symbol mismatch and confusion when multiple symbols are assigned to a single gene. These issues often arise in answers derived from SemMedDB. The UI team tracks these issues. 

Chris B., Andrew S., and Bill B. agreed to use the UI team’s list to explore whether [PubTator](https://www.ncbi.nlm.nih.gov/research/pubtator/) and Text Miner can be leveraged to improve the specificity of SemMedDB answers.

**Comment by andrewsu** [comment]
I think the next step in this line of exploration is to assemble a list of examples where SemMedDB made an error in Named Entity Recognition (NER).  For example, suppose there was a hypothetical abstract in PubMed (PMID:987654) that stated: "A cat clawed the patient's arm resulting in a scratch."  SemMedDB might report a relationship that states `NCBIGene:847 (CAT)` - `causes` - `HP:0032674 (Cutaneous wound)` (with UMLS identifiers, of course).  The relation `A` - `causes` - `B` is correctly extracted, but `A` has been incorrectly linked to the gene instead of the feline. 

Perhaps we can use this issue to track examples like this.  To report an example, please provide three bits of information: the PMID, the incorrectly-identified entity name, and the incorrectly-identified entity CURIE.  So, for the hypothetical example above:

**PMID**: 987654
**Name**: CAT
**CURIE**: NCBIGene:847

**Comment by bill-baumgartner** [comment]
I agree that compiling some examples would be helpful here and should be the next step. Would it be possible to get a dump of the errors that the UI team has compiled?

Diving into the example provided by @karafecho above, this assertion: `human_rabies biolink:associated_with PTPN22` appears in the [9th result from BTE](https://arax.ncats.io/?r=fe05df77-1f7e-4e1e-a422-26e1f429b787) for a query looking for diseases related to a set of genes that includes PTPN22. This assertion was mined from the following sentence from [PMID:32143641](https://pubmed.ncbi.nlm.nih.gov/32143641/): `This has significant implications for achieving the national goal of eliminating dog-mediated human rabies by 2030 and may result in unnecessary costs associated with PEP.`

PEP is listed as a synonym for [PTPN22](https://www.ncbi.nlm.nih.gov/gene/26191). I think there are a number of strategies that could be used to rule out this false positive. The most straightforward would be to make use of the abbreviation definition for PEP that appears earlier in the abstract: `PEP = post exposure prophylaxis`. We could also look into using some machine learning approaches to classify PEP as a particular entity type and hope that it doesn't classify as a gene in this case. More examples would likely help inform the direction(s) we should take to try to resolve false positives such as these.

**Comment by cbizon** [comment]
A few years ago I spent a couple of days looking into how our gene lookup was doing, and how many mixups / false positives we were getting.   The results are in this spreadsheet: https://docs.google.com/spreadsheets/d/1F0ORCvIX80djQXr1kOpMDyXuNqyWs91OIYDRh42fSuo/edit#gid=472403584

it might provide some useful info (admitting that it is very messy, just meant as personal notes)
